### PR TITLE
[FlaxRoberta] Add FlaxRobertaModels & adapt run_mlm_flax.py

### PR DESCRIPTION
**Pull request by patrickvonplaten**
# What does this PR do?

This PR adds all FlaxRobertaModels and adapts `run_mlm_flax.py` to be trainable with FlaxRoberta as well. 

This [roberta-base](https://huggingface.co/patrickvonplaten/norwegian-roberta-base) was pretrained as an example.

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: src/transformers/models/roberta/modeling_roberta.py [hunk 1]
```diff
@@ -121,6 +121,7 @@ def forward(
         if self.position_embedding_type == "absolute":
             position_embeddings = self.position_embeddings(position_ids)
             embeddings += position_embeddings
+
```
Comment (patrickvonplaten) [inline src/transformers/models/roberta/modeling_roberta.py:124]:
```suggestion
```

File: src/transformers/modeling_flax_utils.py [hunk 2]
```diff
@@ -416,7 +416,8 @@ def save_pretrained(self, save_directory: Union[str, os.PathLike], push_to_hub=F
         # save model
         output_model_file = os.path.join(save_directory, FLAX_WEIGHTS_NAME)
         with open(output_model_file, "wb") as f:
-            model_bytes = to_bytes(self.params)
```
Comment (patrickvonplaten) [inline src/transformers/modeling_flax_utils.py:419]: this is required to allows weight saving in distributed mode - see `run_mlm_flax.py`

Comment (sgugger) [reply src/transformers/modeling_flax_utils.py:419]: Like the state_dict argument in the PyTorch equivalent, completely fine by me!

**Review by sgugger — APPROVED** [review]
Thanks a lot for adding those! This looks good to me!

**Pull request merged**